### PR TITLE
Test version 1.0

### PR DESCRIPTION
**Pull request by ksawczak**
Change in the first line

### CODE REVIEW

File: data-store/data-store.markdown [hunk 1]
```diff
@@ -1,6 +1,8 @@
+﻿Test version by ksawczak
```
Comment (Libum) [inline data-store/data-store.markdown]: brawo

**Comment by purifyz** [comment]
SUPER!

File: data-store/data-store.markdown (hunk 1 above)
Comment (purifyz) [inline data-store/data-store.markdown]: @ksawczak @Libum 
brawo

**Comment by ksawczak** [comment]
@Libum Dodane

**Pull request merged**